### DISCUSSION TITLE
Wrong From Version in Commit Body Tables of Some Grouped Dependencies

### DISCUSSION THREAD

**Discussion by Kurt-von-Laven** [renovatebot/renovate#18757] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

We have [commit body tables enabled](https://github.com/ScribeMD/.github/blob/677659df50c2a876d1885f318ec2f73e856770cb/default.json#L11) and have noticed that for some grouped dependencies, the `to` version is incorrectly copied into the `from` column. Both of the incorrect commit body tables we have observed were for grouped dependencies, but this may simply be a coincidence.

### Relevant debug logs

We are using Forking Renovate. See ScribeMD/docker-cache#253 and ScribeMD/docker-cache#258 for examples of PRs with incorrect commit body tables. See ScribeMD/pre-commit-action#160 and ScribeMD/docker-cache#252 for examples of PRs that also bump groups but have correct commit body tables.

### Have you created a minimal reproduction repository?

No reproduction, but I have linked to a public repo where it occurs

**Comment by viceice** [comment, marked as answer]
I don't thing this is a bug. Renovate uses the version from the lockfile as from for the ranges. so it's expected behavior.
